data/blog/2020/2020-03-09-transferring-zf-to-laminas.md

```diff
@@ -0,0 +1,770 @@
+   be updated all at once, instead of having to update them across all
+   repositories.
+
+8. Updated all support files (e.g., `CONTRIBUTING.md`, `CODE_OF_CONDUCT.md`,
+   `SUPPORT.md`).
+
+## Finally: the transition
+
+Finally, after over a year from the first announcement, and 10 months of
+working on the [transfer tool](https://github.com/michalbundyra/laminas-transfer),
+we decided we were ready to launch Laminas, and we chose to do so on the last
+day of 2019.
+
+The days and weeks before, normally holidays for most of us, were spent
+polishing and prepping the tools for the transfer. We even had some "last
+minute changes" which surprised us and prevented us from starting as early
+as we wanted, but managed to migrate everything by around noon UTC time.
+
+We knew not everything was perfect, and there was still a lot left to do, but
+we'd managed to deliver what we'd been promising: we'd deprecated and archived
+all Zend Framework repositories, and created all new components under three
+brand new organisations: [Laminas](https://github.com/laminas),
+[Mezzio](https://github.com/mezzio) and [Laminas API Tools](https://github.com/laminas-api-tools).
+
+### We were right: not everything was perfect
+
+Shortly after completing the migration, we received user reports of issues.
+
+The first the most serious issue was with [namespaced function](#custom-functions).
+We just missed "return" statement in legacy functions with the call to new functions.
+
+Because of that we had to issue patch version for: 
+
+- [laminas-diactoros](https://github.com/laminas/laminas-diactoros),
+- [laminas-stratigility](https://github.com/laminas/laminas-stratigility),
+
+We noticed also that some versions of [laminas-view](https://github.com/laminas/laminas-view)
+had not been transferred correctly: all tags between 2.2.4 and 2.5.3 and we
+have also issued patch version for these.
+
+Also, [Laminas API Tool skeleton application](https://github.com/laminas-api-tools/api-tools-skeleton/)
+had wrong module registered (due to rename from `ZendDeveloperTools` to `Laminas\DeveloperTools`),
+and it had been patched.
+
+> Patched version
+>
+> In above repositories we released new tags with `p1` suffix,
+> as these took precedence during `composer update` operation.
+
+And thankfully that was everything. We've had no blocking issues reported
+since late January, while we continue to get reports of successful migrations.
+
+In last two months, we've also seen many third party repositories migrate to
+Laminas.
+
+If you still have not updated your application or your company is still
+using Zend Framework components, we recommend you migrate, so that you continue
+to get security updates. Please see our [migration guide](https://docs.laminas.dev/migration/).
+
+## Life after life. What next?
+
+We'e had our [first Technical Steering Committee meeting](https://github.com/laminas/technical-steering-committee/blob/master/meetings/minutes/2020-03-02-TSC-Minutes.md),
```

Comment:
I dislike the fragmentation of informations. The link for the meeting should go to the related blog post and the blog post of the meeting should contain the minutes.

Reply:
I consider having the minutes in the blog instead of in the TSC repo, where the agenda and by-laws live, fragmentation.

Reply:
We can create a simple website with some pages which holds this informations. We need also some pages for the documentation writers guide and so on.
I think it is not needed to redirect an end user to a GitHub repository for this kind of informations.